apps/condo/index.js

```diff
@@ -79,35 +81,35 @@ if (!IS_BUILD_PHASE) {
         require('@condo/domains/acquiring/schema'),
     ])
 
+    const { registerTasks } = require('@core/keystone/tasks')
+    // NOTE(pahaz): we need to put it here because it init the readis connection and we don't want it at build time
```

Comment:
we put it here because it inits the redis connection and we don't want it at build time

Reply:
```suggestion
    // NOTE(pahaz): we put it here because it inits the redis connection and we don't want it at build time
```